src/apps/frontend/pages/authentication/otp/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+import constants from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+import useTimer from '../../../utils/use-timer.hook';
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import OTPForm from './otp-form';
+
+export const OTPPage: React.FC = () => {
+  const { startTimer, remaininingSecondsStr, isResendEnabled } = useTimer();
+
+  const navigate = useNavigate();
+
+  const onSuccess = () => {
+    toast.success('Logged in success');
```

Comment:
Do normal login page shows the toast with similar message as well? I haven't seen many applications show a toast on login but rather simply take them to dashboard.

Please remove it from here and login via email as well.